deploy/vagrant/Vagrantfile

```diff
@@ -16,6 +16,10 @@ unless Vagrant.has_plugin?("vagrant-docker-compose")
 end
 
 Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.qemu_use_session = false
+  end
+
```

Comment:
Previously, if a user tries to stand up the vagrant environment in libvirt it would fail because the default for using user sessions does not allow the creation of a bridge device, which is needed by our networking configuration.